### PR TITLE
Adding osp call to cluster-post-config

### PR DESCRIPTION
**Pull request by paigerube14**
Need to be able to call the cluster-post-config in cluster-workers-scaling and properly pass the OSP parameters needed

Passing job: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/paige-e2e-multibranch/job/osp_scale/2/console

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -247,7 +247,17 @@ OPENSHIFT_PROMETHEUS_RETENTION_PERIOD=15d
 OPENSHIFT_PROMETHEUS_STORAGE_SIZE=500Gi
 OPENSHIFT_ALERTMANAGER_STORAGE_SIZE=20Gi
               ''')]
-            } else {
+            } else if(env.VARIABLES_LOCATION.indexOf("osp") != -1){
+              build job: 'scale-ci/e2e-benchmarking-multibranch-pipeline/cluster-post-config', parameters: [
+              string(name: 'BUILD_NUMBER', value: BUILD_NUMBER), string(name: 'HOST_NETWORK_CONFIGS', value:'false'),
```
Comment (nathan-weinberg) [inline Jenkinsfile:252]: Two vars on the same line here
```suggestion
              string(name: 'BUILD_NUMBER', value: BUILD_NUMBER),
              string(name: 'HOST_NETWORK_CONFIGS', value:'false'),
```

**Comment by nathan-weinberg** [comment]
LGTM, but I'll let others comment before merging

**Comment by qiliRedHat** [comment]
1. About the vm type, for AWS, we used
INFRA m5.12xlarge 48vCPU 192GiB
WORKLOAD m5.4xlarge 16vCPU 64GiB

For OSP, the ci.cpu types' memory are small while cpu are big.
INFRA ci.cpu.xxxl 48vCPU 20 GB
WORKLOAD ci.cpu.xxl 36vCPU 20 GB

For OSP, considering we run on 20 nodes on OSP, not as large as AWS (120 nodes), I think the following types may be better.  Surely we have to consider the quota for the types too.
INFRA
ci.m5.xlarge	24	48 GB
WORKLOAD
ci.m5.large	16vCPU	32 GB 


2. I didn't see the following envs passed to OSP.
```
OPENSHIFT_PROMETHEUS_RETENTION_PERIOD=15d
OPENSHIFT_PROMETHEUS_STORAGE_SIZE=500Gi
OPENSHIFT_ALERTMANAGER_STORAGE_SIZE=20Gi
```
While this line of code is run in the [test job](https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/e2e-benchmarking-multibranch-pipeline/job/cluster-post-config/532/console) 
https://github.com/openshift-qe/ocp-qe-perfscale-ci/blob/0cdb1cf87c95fc990e030742426df90cc6903aa9/Jenkinsfile#L407

```
06-29 22:05:50.595  ++ grep vsphere -c
06-29 22:05:50.595  + [[ 0 > 0 ]]
06-29 22:05:50.595  + envsubst
06-29 22:05:50.595  + oc apply -f -
06-29 22:05:54.748  configmap/cluster-monitoring-config created
06-29 22:05:54.748  + oc patch -n openshift-ingress-operator ingresscontrollers.operator.openshift.io default -p '{"spec": {"nodePlacement": {"nodeSelector": {"matchLabels": {"node-role.kubernetes.io/infra": ""}}}}}' --type merge
06-29 22:05:54.748  ingresscontroller.operator.openshift.io/default patched
```
I wonder what's the status of the monitoring pods after the run.
If OSP can support the persistent storage for Monitoring, we should pass the ENVs. If not, OSP should be like the vshpere, moving the monitoring to infra nodes but doesn't configure the persistent storage.

**Comment by paigerube14** [comment]
I think we are going to have to experiment with the sizing of infra and workload nodes to see if we need the CPU or storage more. We could do what you suggested with not moving the storage to the infra/workload nodes but I just wonder if we will notice a difference. Think we need to experiment with large clusters and see how they perform if we leave it as is and then with not moving the storage. Thoughts?

I do not think any of the clouds currently are passing the PROMETHEUS type env vars. I will add those for this PR 

Looking at the configuration sheet it looks like the extra-small and small profiles (3/20) are the same sizing as AWS but there's no information for medium sizing for OSP. Why do you say we are only running up to 20 nodes?

**Comment by paigerube14** [comment]
Properly passing the prometheus variables at the start of the specific cloud types working here: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/paige-e2e-multibranch/job/osp_scale/4/console

**Comment by qiliRedHat** [comment]
> I think we are going to have to experiment with the sizing of infra and workload nodes to see if we need the CPU or storage more. We could do what you suggested with not moving the storage to the infra/workload nodes but I just wonder if we will notice a difference. Think we need to experiment with large clusters and see how they perform if we leave it as is and then with not moving the storage. Thoughts?

I'm worried about the 20G memory, as in bug https://bugzilla.redhat.com/show_bug.cgi?id=2103048, with persistent storage configured, the promethues-k8s pods memory usage grew up to >20G in router-perf test. I guess if the infra nodes have only 20g memory, it could become NotReady.

What do you mean by `with not moving the storage to the infra/workload nodes`?

> Looking at the configuration sheet it looks like the extra-small and small profiles (3/20) are the same sizing as AWS but there's no information for medium sizing for OSP. Why do you say we are only running up to 20 nodes?

Because I saw Roshni planed 20 nodes in our manual scale-ci test https://issues.redhat.com/browse/OCPQE-10768.

**Comment by paigerube14** [comment]
@qiliRedHat I updated the infra and workload node types to what you suggested

You had suggested: `OSP should be like the vshpere, moving the monitoring to infra nodes but doesn't configure the persistent storage.` do we still want to do that or try with the certain node sizes you had suggested and see how it works? 

The jira story you linked is a specific type of osp testing I am supposed to do. We shouldn't have any issues scaling up to 120 nodes just like other cloud types. I thought we had done that configuration before but I can not find the results anywhere.

**Comment by qiliRedHat** [comment]
@paigerube14
`If OSP can support the persistent storage for Monitoring, we should pass the ENVs. If not, OSP should be like the vshpere, moving the monitoring to infra nodes but doesn't configure the persistent storage.`
As you added the persistent storage related ENVs to all clouds, that should not be a problem.

**Pull request merged**